R/coord-transform.r

```diff
@@ -174,3 +178,14 @@ train_trans <- function(scale, limits, trans, name) {
   names(out) <- paste(name, names(out), sep = ".")
   out
 }
+
+# Generate warning when finite values are transformed into infinite values
+#
+# @param old_values A vector of pre-transformation values.
+# @param new_values A vector of post-transformation values.
+# @param axis Which axis the values originate from (e.g. x, y).
```

Comment:
I think it's slightly better to use `#'` and add `@noRd`. 

Would you mind filing an issue or a PR over at https://github.com/tidyverse/style to note that?